ngraph/core/include/ngraph/op/shuffle_channels.hpp

```diff
@@ -62,6 +62,9 @@ namespace ngraph
 
                 int64_t get_axis() const { return m_axis; }
                 int64_t get_group() const { return m_group; }
+                bool evaluate(const HostTensorVector& outputs,
```

Comment:
Please inherit from simple op, not fused op and remove decomposition completely

Reply:
Fixed